<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankCalc</class>
 <widget class="QMainWindow" name="BankCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>690</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(153, 193, 241)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
	background-style: rgb(153, 193, 241);
}</string>
   </property>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_credit_amount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
    <property name="value">
     <double>100000.000000000000000</double>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_term">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1 месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 месяца</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 месяцев</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9 месяцев</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 год</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5 года</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 года</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 года</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 года</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30 лет</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_type">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Дифференцированный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Аннуитетный</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_credit_rate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_annuity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>620</width>
      <height>361</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>310</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <column>
     <property name="text">
      <string>Месяц</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Платеж</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="textEdit_total">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit#textEdit_total {
	background-color: rgb(255, 255, 255);
	border-radius: 10px
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_overpayment">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit#textEdit_overpayment {
	background-color: rgb(255, 255, 255);
	border-radius: 10px
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
	 background-color: #cfcfcf;
     border-radius: 8px;
     font: bold 14px;
     padding: 6px;
	 font-size: 20px;
	border: 1px solid #808080
 }

 QPushButton#pushButton {
     background-color: #ffa000;
 }</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Переплата</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Общая выплата</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cумма кредита</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Процентная ставка %</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cрок</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Тип</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>660</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начисленные проценты</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_annuity_5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>750</y>
      <width>621</width>
      <height>241</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>310</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Сумма вклада</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>660</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма на вкладе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_count_2">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>690</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
	 background-color: #cfcfcf;
     border-radius: 8px;
     font: bold 14px;
     padding: 6px;
	 font-size: 20px;
	border: 1px solid #808080
 }

 QPushButton#pushButton_count {
     background-color: #ffa000;
 }</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>660</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма налога</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>640</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_deposit_2">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>700</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QTextEdit#textEdit_deposit {
     background-color: rgb(255, 255, 255);
	border-radius: 8px
 }</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_tax_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>700</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QTextEdit#textEdit_tax {
     background-color: rgb(255, 255, 255);
	border-radius: 8px
 }</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_percentages_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>700</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QTextEdit#textEdit_percentages {
     background-color: rgb(255, 255, 255);
	border-radius: 8px
 }</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSmartCalc">
    <property name="title">
     <string>CreditCalc</string>
    </property>
    <addaction name="actionSmartCalc"/>
    <addaction name="actionDepositCalc"/>
   </widget>
   <addaction name="menuSmartCalc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSmartCalc">
   <property name="text">
    <string>SmartCalc</string>
   </property>
  </action>
  <action name="actionDepositCalc">
   <property name="text">
    <string>DepositCalc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
